### PR TITLE
doxygen : Add include path

### PR DESCRIPTION
**Pull request by KimDongEon**
To use SmartThings and Audio F/W APIs, you can see which header file to add.

### CODE REVIEW

File: framework/include/st_things/st_things.h [hunk 1]
```diff
@@ -105,7 +113,9 @@ typedef bool (*st_things_set_request_cb)(st_things_set_request_message_s *req_ms
 
 /**
  * @brief Callback registration function for handling request messages.
- * @details The callbacks ensure that a request message will be carried with one of the resource uris from json file of st_things_start().
+ *
+ * @details @b #include <st_things/st_things.h>\n
+ *                      The callbacks ensure that a request message will be carried with one of the resource uris from json file of st_things_start().
```
Comment (jeongchanKim) [inline framework/include/st_things/st_things.h]: why do you use space at the beginning?

File: framework/include/tinyalsa/tinyalsa.h [hunk 2]
```diff
@@ -244,186 +244,205 @@ enum pcm_param {
...
-* @param[in] The device that the pcm belongs to.
-* @param[in] flags Specify characteristics and functionality about the pcm.
-* @param[in] config The hardware and software parameters to open the PCM with.
-* @returns On success, A PCM structure returned. On failure, invalid PCM structure returned.
-* @since Tizen RT v1.1
-*/
+ * @brief Opens a PCM for playback or recording.
+ *
+ * @details @b #include <tinyalsa/tinyalsa.h>
+ * @param[in] The card that the pcm belongs to.
```
Comment (sunghan-chang) [inline framework/include/tinyalsa/tinyalsa.h]: You should write the variable name first and then you should add description of variable.
```
* @param[in] card    The card that the pcm belongs to.
* @param[in] device The device that the pcm belongs to.
* @param[in] flags   Specify characteristics and functionality about the pcm.
```

**Comment by sunghan-chang** [comment]
Please resolve conflicts.

File: framework/include/st_things/st_things.h (hunk 1 above)
Comment (KimDongEon) [reply framework/include/st_things/st_things.h]: modify

File: framework/include/tinyalsa/tinyalsa.h (hunk 2 above)
Comment (KimDongEon) [reply framework/include/tinyalsa/tinyalsa.h]: modify

**Pull request merged**